patch/kernel/rockchip-rk3588-edge/1042-fix-initial-PERST#-GPIO-value.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/drivers/pci/controller/dwc/pcie-dw-rockchip.c b/drivers/pci/controller/dwc/pcie-dw-rockchip.c
```

Comment:
1042*.patch should mean board specific patch. This driver patch is better to start with 0147.